server/matchmaker/algorithm.py

```diff
@@ -1,29 +1,18 @@
-import heapq
-from typing import Dict, Iterable, List, Set
+from typing import Dict, Iterable, List, Set, Tuple
 
 from ..decorators import with_logger
 from .search import Match, Search
 
-################################################################################
-#                           Constants and parameters                           #
-################################################################################
+WeighedGraph = Dict[Search, List[Tuple[Search, float]]]
```

Comment:
`WeightedGraph`? Unless my English is off :)
(The type is consistently without a t, but there is one in `build_weighted`)

Reply:
Yea, that was a typo